examples/benchmark/init.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
@sciabarra, looks like Travis failed as a license header is missing from this file.